app/scripts/controllers/transactions/index.test.js

```diff
@@ -3426,22 +3426,44 @@ describe('Transaction Controller', function () {
       });
     });
 
-    it('ignores new transactions if hash matches existing transaction', async function () {
-      const existingTransaction = TRANSACTION_META_MOCK;
-      const incomingTransaction1 = { ...TRANSACTION_META_MOCK, id: 2 };
-      const incomingTransaction2 = { ...TRANSACTION_META_MOCK, id: 3 };
+    it('ignores new transactions if hash matches existing incoming transaction', async function () {
+      const existingTransaction = {
+        ...TRANSACTION_META_MOCK,
+        id: 1,
+        type: TransactionType.incoming,
+      };
+      const existingTransaction2 = {
+        ...TRANSACTION_META_MOCK,
+        id: 1,
+        type: TransactionType.simpleSend,
+      };
+
+      const incomingTransaction1 = {
+        ...TRANSACTION_META_MOCK,
+        id: 2,
+        type: TransactionType.incoming,
+      };
+      const simpleSendTransaction2 = {
+        ...TRANSACTION_META_MOCK,
+        id: 3,
+        type: TransactionType.simpleSend,
```

Comment:
This shouldn't be possible as all the transactions from this event have the `incoming` transaction type.